analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ImplementSerializationMethodsCorrectly.CSharp11.cs

```diff
@@ -15,9 +15,9 @@ public class GenericAttribute<T> : Attribute { }
     interface IMyInterface
     {
         [OnSerializing]
```

Comment:
Please add a link here to the issue https://github.com/SonarSource/sonar-dotnet/issues/6331.  Also, I am not sure it should be marked simply as Noncompliant. 
I suggest adding a link to these tests in the aforementioned issue and when that one gets implemented the UTs will get updated accordingly.